src/main/resources/META-INF/plugin.xml

```diff
@@ -8,17 +8,12 @@
     <!-- add java functionality. See https://blog.jetbrains.com/platform/2019/06/java-functionality-extracted-as-a-plugin/ for more information -->
     <depends>com.intellij.modules.java</depends>
 
-    <description>
-        <![CDATA[
-            <p>A plugin that uses EvoSuite to generate tests. Developed by TU Delft students.
```

Comment:
Why did you remove these lines? It was just a placeholder of now

Reply:
I was getting warnings during the build process from the `patchPluginXml` gradle task that I couldn't fix so I removed it for now.

Reply:
Alright